### ISSUE TITLE
Alert dialog overlay

### ISSUE THREAD

**Issue by roblevintennis** [morkro/vue-a11y-dialog#38]
I think I may have made a mistake in the alert dialog implementation for alert dialog role case…

See:
https://github.com/morkro/vue-a11y-dialog/blob/main/src/A11yDialog.vue#L12

and 
https://github.com/morkro/vue-a11y-dialog/blob/main/src/A11yDialog.vue#L12

Notice we only omit the close handler but not the attribute.

And then if we read @KittyGiraudel  instructions for this:
https://a11y-dialog.netlify.app/advanced/alert-dialog

I suppose the fix might look like `:data-a11y-dialog-hide="role === 'alertdialog' ? undefined : true"` or in context:

```html
    >
      <div
        :data-a11y-dialog-hide="role === 'alertdialog' ? undefined : true"
        tabIndex="-1"
        :class="classNames.overlay"
        @click="role === 'alertdialog' ? undefined : close"
      />
```

I'm happy to make the change next spare cycle I get @morkro

**Comment by KittyGiraudel** [comment]
Yep, seems that this needs to be fixed. :)

**Comment by roblevintennis** [comment]
Ok, I've made an issue in my own repo at https://github.com/AgnosticUI/agnosticui/issues/137 so I remember to do address this next week. Thanks for confirming.

**Comment by roblevintennis** [comment]
https://github.com/morkro/vue-a11y-dialog/pull/39 fixes this issue.

**Comment by morkro** [comment]
Thanks @roblevintennis—released `1.1.2` :)
